pkg/landscaper/installations/installation.go

```diff
@@ -7,27 +7,35 @@ package installations
 import (
 	"fmt"
 
-	lsv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
 	lsv1alpha1helper "github.com/gardener/landscaper/apis/core/v1alpha1/helper"
+
+	lsv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
 	"github.com/gardener/landscaper/pkg/landscaper/blueprints"
 )
 
-// Installation is the internal representation of a installation
-type Installation struct {
-	Imports   map[string]interface{}
-	Info      *lsv1alpha1.Installation
-	Blueprint *blueprints.Blueprint
+type InstallationBaseInterface interface {
```

Comment:
not sure if we really need the interface.
I think just having the `InstallationBase` struct is enough.

Reply:
We introduced the interface to be able to provide Installations and InstallationBases to methods which only need objects fulfilling the contract of InstallationBaseInterface. Of course we could go another way by removing the interface and changing the contract of those functions to InstallationBase, but then we must convert Installations first into InstallationBases when these are called.

Reply:
we removed the interface